### PR TITLE
[WFLY-12904] adding quickstart for ejb txn remote calls from server to server

### PR DESCRIPTION
**Pull request by ochaloup**
https://issues.redhat.com/browse/WFLY-12904

A high level overview of the quickstart that will be added regarding of the remote EJB server to server propagation.

### CODE REVIEW

**Comment by ochaloup** [comment]
@mjurc if you have some time it would be nice if may review this :-) thanks.

File: openshift/WFLY-12904.adoc [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+* https://issues.redhat.com/browse/WFLY-12904[WFLY-12909]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/CLOUD-2262[CLOUD-2262]
+
+=== Dev Contacts
+
+* mailto:{ochaloup@redhat.com}[{author}]
```
Comment (simkam) [inline openshift/WFLY-12904.adoc]:
```
mailto:ochaloup@redhat.com[{author}]
```

File: openshift/WFLY-12904.adoc [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+=== QE Contacts
+
+* mailto:{msimka@redhat.com}[msimka@redhat.com]
```
Comment (simkam) [inline openshift/WFLY-12904.adoc]: mailto:msimka@redhat.com[msimka@redhat.com]

File: openshift/WFLY-12904.adoc [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+Both EJBs (on "client server" and on the second server)
+need to process some "transactional" work (which is e.g. inserting data to a database)
+for transaction manager may run XA transactions.
+
+The way of running the transaction work is about to be investigated.
+As the goal is that this quickstart would be run not only on standalone servers
+but mainly then focused for OpenShift showcase the type of resource
+(database, JMS, mock XA resource) is important for having an easy way how to
+get it to the OpenShift docker image.
```
Comment (simkam) [inline openshift/WFLY-12904.adoc]: Could you please reword this sentence?

Comment (ochaloup) [reply openshift/WFLY-12904.adoc]: Thanks for the feedback. Please, check if it's better now.

Comment (simkam) [reply openshift/WFLY-12904.adoc]: looks good, thanks

**Comment by ochaloup** [comment]
@bstansberry do you think this proposal PR could be merged in this per https://issues.redhat.com/browse/EAP7-1358?

**Comment by ochaloup** [comment]
@jmesnil @kabir do you think this could be merged? It's for quickstart https://github.com/wildfly/quickstart/pull/372 which should  be going to WFLY 23.

**Pull request merged**